### PR TITLE
Fixed bug in IEEEXploreFetcher

### PR DESCRIPTION
**Pull request by oscargus**
Fixed a bug which sometimes stalled the search, leaving some entries not parsed.
Changed the earlier print out logging/debugging to use the logging system.
Code clean up.

### CODE REVIEW

File: src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java [hunk 1]
```diff
@@ -635,13 +622,11 @@ private BibtexEntry parseNextEntry(String allText, int startIndex) {
...
             if (typeMatcher.find()) {
                 typeName = typeMatcher.group(1);
-                if (typeName.equalsIgnoreCase("IEEE Journals &amp; Magazines") || typeName.equalsIgnoreCase("IEEE Early Access Articles") ||
-                        typeName.equalsIgnoreCase("IET Journals &amp; Magazines") || typeName.equalsIgnoreCase("AIP Journals &amp; Magazines") ||
-                        typeName.equalsIgnoreCase("AVS Journals &amp; Magazines") || typeName.equalsIgnoreCase("IBM Journals &amp; Magazines") ||
-                        typeName.equalsIgnoreCase("TUP Journals &amp; Magazines") || typeName.equalsIgnoreCase("BIAI Journals &amp; Magazines")) {
+                if (typeName.equalsIgnoreCase("IEEE Journals &amp; Magazines") || typeName.equalsIgnoreCase("IEEE Early Access Articles") || typeName.equalsIgnoreCase("IET Journals &amp; Magazines") || typeName.equalsIgnoreCase("AIP Journals &amp; Magazines") || typeName.equalsIgnoreCase("AVS Journals &amp; Magazines") || typeName.equalsIgnoreCase("IBM Journals &amp; Magazines") || typeName.equalsIgnoreCase("TUP Journals &amp; Magazines") || typeName.equalsIgnoreCase("BIAI Journals &amp; Magazines")) {
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java]: @koppor @stefan-kolb  @matthiasgeiger  this seems to be an issue with wrong formatting as this line is way too long.

**Comment by simonharrer** [comment]
:+1:

**Comment by simonharrer** [comment]
Can you please rebase this pull request on master? In the present state it cannot be merged it directly.

File: src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java (hunk 1 above)
Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java]: I already pointed that out somewhere... 77d1ea868c3381d78b19f5e15a1364de9033eb2a introduces this behavior... @koppor do you remember why you've changed this?

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [master]
+	- Fixed a bug in the IEEEXploreFetcher
```
Comment (stefan-kolb) [inline CHANGELOG]: Again spaces instead of a tab.

**Comment by oscargus** [comment]
OK! Will see if I can figure out how to rebase it (I thought I had found a good practice for this) and change the tabs to spaces in ChangeLog.

The if statement does indeed look quite ugly... I'll hand edit it for now.

**Comment by oscargus** [comment]
Ahh should have rebased... Should I start over or can it be solved in some clever way?

**Comment by koppor** [comment]
Merge is also OK. For me, it's simpler to use than resolving conflicts at rebasing :)

**Pull request merged**